lib/steep/drivers/checkfile.rb

```diff
@@ -105,6 +105,7 @@ def run
           steepfile: project.steepfile_path,
           args: [],
           delay_shutdown: true,
+          severity_level: nil,
```

Comment:
It's also better to add the `--severity-level` option to checkfile subcommand too. I'll work it after this PR merged.